test/e2e/adminactions_kubernetesobjects.go

```diff
@@ -0,0 +1,123 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var _ = Describe("Admin actions", func() {
+	BeforeEach(runAdminTestsInDevOnly)
+
+	Specify("kubernetesobjects list", func() {
+		result, err := adminRequest("GET", "kubernetesobjects", "", nil, "kind=configmap", "namespace=openshift-machine-api")
+		Expect(err).NotTo(HaveOccurred())
+		obj := &unstructured.Unstructured{}
+		err = obj.UnmarshalJSON(result)
+		Expect(err).NotTo(HaveOccurred())
+
+		objs, err := obj.ToList()
+		Expect(err).NotTo(HaveOccurred())
+
+		var validNames = []string{
+			"cluster-autoscaler-operator-ca",
+			"cluster-autoscaler-operator-leader",
+			"kube-rbac-proxy",
+			"kube-rbac-proxy-cluster-autoscaler-operator",
+			"machine-api-operator",
+			"machine-api-operator-images",
+		}
+
+		// get objects with kubeclient and compare
+		cm, err := Clients.Kubernetes.CoreV1().ConfigMaps("openshift-machine-api").Get("cluster-autoscaler-operator-leader", metav1.GetOptions{})
```

Comment:
I think a get here is a bit odd. Why not do a List intsead and populate validNames from the result.